src/index.js

```diff
@@ -2,14 +2,22 @@ import React from "react";
 import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
+import Luggage from "./pages/luggage";
 import { makeServer } from "./server";
+import { BrowserRouter } from "react-router-dom";
+
+import { CartContext } from "./context/cart-context.js";
 
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <BrowserRouter>
+      <CartContext.Provider value={{ items: 5 }}>
```

Comment:
You can also create a component that returns this context. Something like `<CartProvider></CartProvider>`

Reply:
Thanks for the comment siddhant, I'll update the component, when I'll use this context for real functionality.
It was for simply testing it